### ISSUE TITLE
ability to write metadata into exif tags

### ISSUE THREAD

**Issue by jywarren** [publiclab/image-sequencer#169]

**Comment by jywarren** [comment]
[GitMate.io](https://gitmate.io) thinks a possibly related issue is #118: Assorted improvements to be broken into separate issues.

**Comment by ccpandhare** [comment]
This would be pretty cool!

**Comment by ccpandhare** [comment]
Possible implementation modules:
- https://www.npmjs.com/package/piexifjs

**Comment by tech4GT** [comment]
@jywarren can you please elaborate on this one a little?

**Comment by jywarren** [comment]
This was was just to try to come up with a standard namespace and
convention for the metadata generated by all the steps, and to write that
into a custom EXIF tag in the image, as you go. That way, there is some
"trace" of how the image was produced via the steps. Does that make sense?
Sort of an evidence trail, or showing your work.

**Comment by tech4GT** [comment]
@jywarren Yeah I get this!! So we are embedding the sequence data in the image itself, right?

**Comment by jywarren** [comment]
yep!
